db/migrate/20160112133345_add_type_to_lookup_value.rb

```diff
@@ -0,0 +1,11 @@
+class AddTypeToLookupValue < ActiveRecord::Migration
+  def change
+    add_column :lookup_values, :type, :string
+    LookupValue.reset_column_information
+    LookupValue.all.each do |lookup_value|
```

Comment:
This shouldn't be in a `change` method as it won't run backwards, it should have separate `up`/`down` methods.